.agents/skills/webawesome/references/components/format-date.md

```
@@ -0,0 +1,119 @@
+
+To import this component for React 18 or below, use the following code:
+
+```js
+import WaFormatDate from '@awesome.me/webawesome/dist/react/format-date/index.js';
+```
+
+### Attributes & Properties
+
+| Name | Description | Reflects |
+| --- | --- | --- |
+| \`date\` date | \`date.toISOString()\` The date/time to format. If not set, the current date and time will be used. When passing a string, it's strongly recommended to use the ISO 8601 format to ensure timezones are handled correctly. To convert a date to this format in JavaScript, use . Type Date \\| string Default new Date() | |
```

markdownlint-cli2 (0.23.2)

[warning] Table column count
Expected: 3; Actual: 4; Too many cells, extra data will be missing

(MD056, table-column-count)